pkg/config/cmd/cmd.go

```diff
@@ -30,7 +30,7 @@ func NewPrintNameOrErrorAfter(out, errs io.Writer) func(*resource.Info, error) {
 // Create attempts to create each item generically, gathering all errors in the
 // event a failure occurs. The contents of list will be updated to include the
 // version from the server.
-func (b *Bulk) Create(list *kapi.List, namespace string) []error {
+func (b *Bulk) Create(list *kapi.List, namespace string) bool {
```

Comment:
we should return error list so we can inspect the individual errors (dunno if that is happening right now).